contracts/test/strategies/fraxeth.fork-test.js

```diff
@@ -24,7 +24,7 @@ describe("ForkTest: FraxETH Strategy", function () {
   });
 
   describe("Mint", function () {
-    it.skip("Should allow mint with frxETH", async () => {
+    it("Should allow mint with frxETH", async () => {
```

Comment:
The names of these should change to "Should not allow" for the ones that are testing the revert case.

Reply:
Ah, yes